modyn/supervisor/internal/grpc_handler.py

```diff
@@ -485,27 +511,18 @@ def seed_selector(self, seed: int) -> None:
 
         assert success, "Something went wrong while seeding the selector"
 
-    def start_evaluation(self, trained_model_id: int, pipeline_config: dict) -> dict[int, EvaluationStatusTracker]:
+    def start_evaluation(self, model_id: int, pipeline_config: dict) -> dict[int, EvaluationStatusTracker]:
         if not self.connected_to_evaluator:
             raise ConnectionError("Tried to start evaluation at evaluator, but there is no gRPC connection.")
 
```

Comment:
Obligatory comment: Please double check before merging that a full pipeline works

Reply:
I performed full pipeline execution. Everything seems to work fine :)

However, I had to do some small changes to the supervisor (in `wait_for_new_data`). Something with the types didn't match up. Maybe because we introduced stopwatch?

Reply:
Hmm, I am not sure what the problem with the types was... not sure why we havent faced that before